apps/domain/priv/repo/migrations/20230425082110_create_actors.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Domain.Repo.Migrations.CreateActors do
+  use Ecto.Migration
+
+  def change do
+    create table(:actors, primary_key: false) do
```

Comment:
`first_name` `last_name`?

Reply:
Do we want to require first and last names to be not empty at all times? (I guess yes otherwise it can be a bit messy to figure out who is who.) 

Also, do we want people to edit their first and last names when they are synced from IdP? (I think no because we don't want Andrew to rename himself Jamil and do some social engineering call).

Reply:
Hm, good question. When `actors` are `users`, yes, we should require those. When `actors` are `api` or `service_account` then it wouldn't make sense to require them.

Also, did you have an idea for how multiple `identities` will map to the same `actor`? I think `email` is probably our best candidate for now unfortunately. In the future maybe we can allow the admin to define attribute mappings from IdP -> Firezone. (I think Azure is the only misbehaved major IdP here with its User Principal Name stuff)

Reply:
> Also, did you have an idea for how multiple identities will map to the same actor?
We don't map them automatically. There will be multiple identities when you already have one and then connect IdP (or if we build UI to add email even if you have IdP connected), but there will be no logic around using some fields to deduplicate actors - it's error-prone, rare for orgs and admins should do it.